components/script/dom/htmlformelement.rs

```diff
@@ -455,7 +455,27 @@ impl HTMLFormElement {
         // Step 1-3
         let _unhandled_invalid_controls = match self.static_validation() {
             Ok(()) => return Ok(()),
-            Err(err) => err
+            //Err(err) => err,
+            Err(err) => {
+                match err[0] {
+                    FormSubmittableElement::InputElement(ref i) =>
+                        println!("First element didn't pass: {}",
```

Comment:
Stdout is all we've got at the moment. We should be calling `Focus()` on the element, however.